srml/timestamp/src/lib.rs

```diff
@@ -116,6 +171,7 @@ impl ProvideInherentData for InherentDataProvider {
 			})
 	}
 
+	/// Tries to convert the `InherentError` into a string
```

Comment:
Indeed I'm in favor of documentating all public stuff even obvious ones, but on the other end I'm not necessarily in favor of documentating private function when name is expressful enough